server/src/test/java/io/spine/server/procman/ProcessManagerRepositoryTest.java

```diff
@@ -346,9 +348,10 @@ void events() {
                     .comparingExpectedFieldsOnly()
                     .isEqualTo(event.messageId());
             PmProjectStarted eventMessage = (PmProjectStarted) event.enclosedMessage();
-            assertThat(unpack(systemEvent.getEntity().getId()))
-                      .comparingExpectedFieldsOnly()
-                      .isEqualTo(eventMessage.getProjectId());
+            assertThat(unpack(systemEvent.getEntity()
+                                         .getId()))
+                    .comparingExpectedFieldsOnly()
+                    .isEqualTo(eventMessage.getProjectId());
```

Comment:
Why do we compare only the IDs here and below? Can we just hand-craft the expected message and compare the whole message instead?